### DISCUSSION TITLE
The newly deleted data will appear in the query

### DISCUSSION THREAD

**Discussion by Zhao-Panpan** [aio-libs/aiomysql#716] [Q&A] [answered]
I'm using aiomysql 0.0.20. I'm using a connection pool.When I call the API to delete a piece of data, I use the API to query the database. The data obtained is unstable, and the deleted data will be displayed.
What is the cause of this?

I hope I can get your help. Thank you.

**Comment by Nothing4You** [comment, marked as answer]
Hi,

unfortunately this question is quite old and therefore probably not relevant anymore.
I suspect that you used the pool with the default of `autocommit=False` which may have caused this.
Without more details (e.g. a reproducible code example) I don't think there's anything to go on from this.
